yapapi/runner/__init__.py

```diff
@@ -488,6 +512,12 @@ class TaskStatus(Enum):
 
 
 class Task(Generic[TaskData, TaskResult], object):
+    """One computation unit.
+
+    Represents one computation unit that will be run on the provider
+    (e.g. rendering of one frame); should contain all data needed
+    by the requestor to prepare command list for the provider.
```

Comment:
Let's just end this comment on `;` (replacing `;` with `.` of course).

Reply:
Fixed.